### PR TITLE
chore: query issue fix

### PR DESCRIPTION
**Pull request by aziyatali**

### CODE REVIEW

**Comment by JonghunYu** [comment]
@aziyatali adding refetch function doesn't seem like a good idea. Please check `useQuery` hook executes the initial data loading or not first.

**Comment by aziyatali** [comment]
@JonghunYu  Got it

File: __test__/kross-client/loan.tsx [hunk 1]
```diff
@@ -41,10 +41,10 @@ export const loan = () => {
         wrapper,
       }
     );
-    await waitFor(() => expect(result.current.isSuccess).toBe(true), {
-      timeout: 30000,
+    await waitFor(() => {
```
Comment (Jscripter-pk) [inline __test__/kross-client/loan.tsx:44]: At every point It is kind of necessary to check first is isSuccess is true then go for checking data

File: __test__/kross-client/loan.tsx [hunk 2]
```diff
@@ -60,10 +60,11 @@ export const loan = () => {
         wrapper,
       }
     );
-    await waitFor(() => expect(result.current.isSuccess).toBe(true), {
-      timeout: 30000,
+
+    await waitFor(() => {
```
Comment (Jscripter-pk) [inline __test__/kross-client/loan.tsx:64]: And also it is better to just do expect for result.current.data to be defined instead of first destructuring it

**Pull request merged**